dll/cpl/desk/lang/pt-PT.rc

```diff
@@ -185,7 +187,7 @@ FONT 8, "MS Shell Dlg"
 BEGIN
     GROUPBOX "Visualização", -1, 7, 6, 237, 56, WS_GROUP
     //LTEXT "Se a resolução do ecrã tornar os items demasiado pequenos para ver confortavelmente, pode aumentar o valor ppp para compensar. Para alterar apenas o tamanho do tipo de letra, clique em ""Cancelar"" e vá até ao separador ""Aspecto"".", -1, 14, 20, 220, 45
```

Comment:
Why is that line commented out? Is that also like that in other languages? Should the line be stripped or the comment be removed?

Reply:
I don't know, it wasn't me.
This text exists in win2003 Srv, and is of great help to the user.
Can I reset visible text?